posthog/api/organization.py

```diff
@@ -80,6 +81,8 @@ class Meta:
             "available_features",
             "domain_whitelist",
             "is_member_join_email_enabled",
+            "only_allowed_team_ids",
```

Comment:
I think this might be better computed by using the effective level for each team inside the `teams` attribute. It's less bloated, one single source of truth, and less queries and less computation in a critical endpoint.